src/MaxText/configs/base.yml

```diff
@@ -287,6 +287,13 @@ use_post_attn_norm: False
 use_post_ffw_norm: False
 mla_naive_kvcache: True
 
+
+# Adding Mixture of Block Attention Support (MoBA): https://github.com/MoonshotAI/MoBA/blob/master/MoBA_Tech_Report.pdf
+# Flash attention with moba is not supported currently
```

Comment:
No need to write comment here, let's assert it in `pyconfig.py`, somewhere here: https://github.com/AI-Hypercomputer/maxtext/blob/95542d91557521fd66b38e343d34d24ec94d4a50/src/MaxText/pyconfig.py#L169


```
def validate_moba_attention(keys):
  if keys["moba"] and keys["attention"] in ("autoselected", "flash", "cudnn_flash_te", "cudnn_flash_jax", "paged"):
      raise ValueError("MoBA is only supported dot_product attention") 
```